website/docs/tutorial/getting-set-up/setting-up-databricks.md

```diff
@@ -85,69 +85,63 @@ Congrats! You are now ready to start working in the workspace.
 <Lightbox src="/img/databricks_tutorial/images/welcome_page.png" title="Welcome to the Databricks Workspace" />
 </div>
 
-Next up, you will need a cluster for our training. A spark cluster is going to be your compution resource for running your analytics workloads. We will be using it to run our queries. 
-
-12. Hovering over the Databricks logo on the left, you will see a left hand side console. Click **Compute** and you will see the clusters at your disposal. For this tutorial, you will be using the starter cluster that comes with our workspace but you can always create your own. 
-
-13. Hover your pointer over the space under `Actions` of your starter cluster, click on the play button. Confirm that you do want to start up the cluster.'
-
-<div style={{maxWidth: '400px'}}>
-<Lightbox src="/img/databricks_tutorial/images/start_cluster.png" title="Start the Cluster" />
-</div>
-
-If you ever want to edit the cluster in terms of the name of the cluster and resources allocated, you can click on the cluster name and the `Edit` button. Please note that you cannot edit the cluster when it is starting up or running.
-
 ## Loading data
 
-Our next step is to load some data to transform. Luckily for us, Databricks makes it really easy for us to upload data. 
+Our next step is to load some data to transform. Luckily for us, Databricks makes it really easy for us to upload data.
+
+1. First we need a SQL endpoint. Find the drop down menu and toggle into the SQL space.
+    <div style={{maxWidth: '400px'}}>
+    <Lightbox src="/img/databricks_tutorial/images/go_to_sql.png" title="SQL space" />
+    </div>
+2. We will be setting up a SQL endpoint now.  Select `SQL Endpoints` from the left hand side console.  You will see that a default SQL Endpoint exists.  
+    <div style={{maxWidth: '400px'}}>
+    <Lightbox src="/img/databricks_tutorial/images/sql_endpoints.png" title="SQL Endpoints" />
+    </div>
+3. Click **Start** on the Starter Endpoint.  This will take a few minutes to get the necessary resources spun up.
 
-1. Download the three CSV files locally that you will need for this tutorial. You can find them here:
+4. While you're waiting, download the three CSV files locally that you will need for this tutorial. You can find them here:
     - [jaffle_shop_customers.csv](http://dbt-tutorial-public.s3-us-west-2.amazonaws.com/jaffle_shop_customers.csv)
```

Comment:
I'm having trouble downloading from this link. Nothing happens. But when I copy and past the URL into a new tab, I get the download. I'm using Chrome.  Should it be `https`?

Reply:
Yup! You're correct; I just fixed it.